doc/reference/networking/net_timeout.rst

```diff
@@ -10,10 +10,50 @@ Network Timeout
 Overview
```

Comment:
The commit subjects in these commits have redundant information, we can just say `net: timeout: ....` and `net: if: ....` so no neeed to have extra `net` here.